src/datafactory/setup.py

```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.1.0'
+VERSION = '0.6.0'
```

Comment:
This version is overwritten in https://github.com/Azure/azure-cli-extensions/blob/main/src/datafactory/azext_datafactory/manual/version.py#L11

Can you please update there instead?

Reply:
Updated